TransferMethod/Sources/ListTransferMethodController.swift

```diff
@@ -34,7 +34,6 @@ final class ListTransferMethodController: UITableViewController {
                                                                               .localized())
     private lazy var addAccountButton: UIButton = view.setUpEmptyListButton(text: "add_account_title".localized(),
                                                                             firstItem: emptyListLabel)
-
```

Comment:
Why this change?

Reply:
Added the empty space. This happens when i added and removed the constant(Pagename , Pagegroup)